src/Attachment.tsx

```diff
@@ -153,6 +163,20 @@ export const AttachmentView = (props: {
                 </div>
             );
 
+        // only supported for 'skype' channel.
```

Comment:
This comment doesn't really make sense for WebChat (which is specific to the DirectLine channel). Can you clarify that this type is intended for the Emulator for testing Skype-specific bots?